t/helper/test-quote.c

```diff
@@ -0,0 +1,28 @@
+#include "test-tool.h"
+#include "quote.h"
+#include "strbuf.h"
+#include "string.h"
+
+int cmd__quote_buf_pretty(int argc, const char **argv)
+{
+	struct strbuf buf_payload = STRBUF_INIT;
+
+	if (!argv[1]) {
+		strbuf_release(&buf_payload);
+		die("missing input string");
+	}
+
+	if (!strcmp(argv[1], "nullString"))
+		sq_quote_buf_pretty(&buf_payload, NULL);
+
```

Comment:
Nit: Remove the empty line above the `else if` and the `else` below?  Elsewhere in the code I'm not seeing blank lines between these.

Reply:
:) I just got back from asking Jeff about this and he said that it is fine both ways. I am going to float this in with the new lines to see if it gets through (I like it like that)